features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/edition/RoomDetailsEditionNode.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.features.roomdetails.impl.edition
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import com.bumble.appyx.core.modality.BuildContext
+import com.bumble.appyx.core.node.Node
+import com.bumble.appyx.core.plugin.Plugin
+import dagger.assisted.Assisted
+import dagger.assisted.AssistedInject
+import io.element.android.anvilannotations.ContributesNode
+import io.element.android.libraries.di.RoomScope
+
+@ContributesNode(RoomScope::class)
+class RoomDetailsEditionNode @AssistedInject constructor(
```

Comment:
Yeah, it's not proper English.

AFAIK there's not a specific noun for it in English. If you "add" something it's an "addition", but if you "edit" something it's... an "edit". Which makes the name a bit clunky because it's not obvious if it's a verb or a noun.

I left it as-is because it seemed to make sense even if it's not a real word, I couldn't think of a better alternative, and it amused me a little :laughing: 

Do you think it's confusing? Or you easily understood what it meant?

Reply:
In my mind it clashed with the actual meaning of "Edition" (i thought of a book :) ). But perhaps it's just me.
Personally I'd prefer `RoomDetailsEditNode`. But I understand this can be very subjective so we can ask for more input here if you'd like.

Reply:
`RoomDetailsFormNode` ? (my 2 cents)

Reply:
I've gone with `Edit` for now. `Form` makes me think of the screen at the end of the create room flow, where as `Edit`'s purpose is unambiguous.

Reply:
Sorry for my bad English, `edition` is correct in French... 😅